devops/Makefile

```diff
@@ -200,6 +203,7 @@ init-secrets:
 		-p S3_ENDPOINT=$(S3_ENDPOINT) \
 		-p S3_DEFAULT_BUCKET=$(S3_DEFAULT_BUCKET) \
 		-p S3_REGION=$(S3_REGION) \
+		-p QUEUE_DASHBOARD_BASE_URL=$(HOST) \
```

Comment:
Adding comment here for record. As we discussed, need to validate if we have to prefix the `https://`before the HOST to make it work.

Reply:
Prefixed the host with "https" to ensure the URL will be absolute.